models/migrations/v27.go

```diff
@@ -39,13 +34,36 @@ func convertIntervalToDuration(x *xorm.Engine) (err error) {
 		return err
 	}
 
+	dialect := x.Dialect().DriverName()
+
+	switch dialect {
+	case "mysql":
+		_, err = sess.Exec("ALTER TABLE mirror MODIFY `interval` BIGINT")
+	case "postgres":
+		_, err = sess.Exec("ALTER TABLE mirror ALTER COLUMN `interval` SET DATA TYPE bigint")
```

Comment:
I dunno if XORM takes care of this @lunny but PostgreSQL quotes for identifiers should be doublequote, not backtick.